### ISSUE TITLE
Quest: Lost in Battle

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#83]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

The Barrens / Southern Barrens / NPC: Beaten Corpse / Quest: Lost in Battle
Link: https://www.wowhead.com/wotlk/quest=4921/lost-in-battle

- The Barrens / Southern Barrens / NPC: Beaten Corpse
  Link: https://www.wowhead.com/wotlk/npc=10668/beaten-corpse

  Greeting: despite the voice-over is perfect, its describes a sequence of actions. The voice of the narrator is more suitable in this case.

  "I inspect the body further." text: despite the voice-over is perfect, its describes a sequence of actions. The voice of the narrator is more suitable in this case.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_VanillaExtra_v1.5.
